mne_bids/write.py

```diff
@@ -283,11 +283,16 @@ def _participants_tsv(raw, subject_id, fname, overwrite=False,
 
     subject_age = "n/a"
     sex = "n/a"
+    hand = 'n/a'
     subject_info = raw.info['subject_info']
     if subject_info is not None:
         sexes = {0: 'n/a', 1: 'M', 2: 'F'}
         sex = sexes[subject_info.get('sex', 0)]
 
+        # add handedness
+        hand_options = {0: 'n/a', 1: 'R', 2: 'L', 3: 'A'}
```

Comment:
did you invent these options? according to https://mne.tools/dev/generated/mne.Info.html only 1 and 2 are possible options

Reply:
see discussion here @jasmainak https://github.com/mne-tools/mne-bids/pull/354#discussion_r382652813